### PR TITLE
[stable/yugaware] auto-create imagepullsecret

### PR DESCRIPTION
**Pull request by aegershman**
In order to not have to manually create the imagePullSecret in the same namespace, the imagePullSecret dockerconfigjson should be able to be defined inline in a values.yml ([or an encrypted `secrets.yaml`](https://github.com/roboll/helmfile#secrets)) supplied with the chart itself.

Alternatively this could be defined under a `secrets:` block, something like this:

```yml
image:
  pullSecret: foo

...

secrets:
  create: true
  imagePullSecret: xyzabc123
```

### CODE REVIEW

**Comment by ramkumarvs** [comment]
@aegershman LG. only thing i need to run it by internally is the chart version. I can merge this change after that

File: stable/yugaware/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: 1.3.2.0-b18
-version: 1.3.2
+version: 1.3.3
```
Comment (ramkumarvs) [inline stable/yugaware/Chart.yaml:3]: change this to include `prerelease` tag

**Review by ramkumarvs — APPROVED** [review]
Great change, @Arnav15 we should try to bundle this into our YW provider config as well

File: stable/yugaware/Chart.yaml (hunk 1 above)
Comment (aegershman) [reply stable/yugaware/Chart.yaml:3]: so should this be `1.3.3-prerelease-1`?

Comment (ramkumarvs) [reply stable/yugaware/Chart.yaml:3]: yup

**Comment by aegershman** [comment]
hello from the future! hope all is well! closing to prune dead PRs

**Pull request closed without merging**